src/main/java/ch/njol/skript/expressions/ExprScoreboardTags.java

```diff
@@ -0,0 +1,143 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.coll.CollectionUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Scoreboard Tags")
+@Description({"Scoreboard tags are a simple list of single-word texts stored directly in the data of an entity.",
```

Comment:
A better description would be:
```
An expression for setting an changing the scoreboard tags of an <a href='classes.html#entity'>entity</a>. More info about scoreboard tags can be found at <wiki link>.

You can also use the <a href='conditions.html#CondHasScoreboardTag'>has scoreboard tag</a> condition to check whether an entity has the given tag.

Note: this only works on 1.11 and up since Spigot added support for it on that version.```

Reply:
"setting and changing"? and this info should be provided: "added in 1.9 to the game"

Reply:
Actualy i don't like your description that much. I see mine better and descriptive. But i will edit mine a bit..

By the way `has scoreboard tag` should be capitalized becase it is not related to the current sentence, it is a title that refers to something other. I hope you understand me lol.

Reply:
I mean, mine was more of a template for you to implement it, you can change it as the current one doesn't reflect what the expression does exactly.

You can get the idea from mine and change it as you want.